docs/linux/zbd-api.md

```diff
@@ -86,16 +79,13 @@ This attribute value is always 0 for a regular block device.
 ## *ioctl()* Application Programming Interface
 
 The C header file `/usr/include/linux/blkzoned.h` contains macro definitions and
-data structure definitions allowing an application developer to obtain
-information on a zoned block device and to manage the zones of the device.
+data structure definitions that allow application developers to obtain
+information about zoned block devices and to manage the zones of the devices.
 
 ### Zone Information Data Structures
 
-The data structure `struct blk_zone` defines a zone descriptor structure which
-completely describes a zone information: the zone location on the device, the
-zone type, its condition (state), and the position of the zone write pointer
-for sequential zones. Up to kernel version 5.8, this data structure is as shown
-below.
+The data structure `struct blk_zone` defines a zone-descriptor structure that 
+contains a complete description of a zone's information: this includes the zone's location on the device, the zone type, its condition (state), and the position of the "zone write pointer" (for sequential zones). For kernels Up to kernel version 5.8, this data structure is as shown below.
```

Comment:
Minor nit: "...contains a complete description of a zone's information" -> "...contains a complete description of a zone" ?

Reply:
Also, I would not quote "zone write pointer". I do not see why we should quote it.